fastestimator/backend/_set_lr.py

```
@@ -12,17 +12,14 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 # ==============================================================================
-from typing import Optional, Union
+from typing import Optional
 
 import torch
 
-from fastestimator.backend._get_lr import get_lr
-
 
 def set_lr(model: torch.nn.Module, lr: float, weight_decay: Optional[float] = None):
```

Ruff (0.14.3)

Unused function argument: `weight_decay`

(ARG001)